pack.ps1

```diff
@@ -15,6 +15,9 @@ $ErrorActionPreference = "Stop"
 $buildDirectoryPath = "$basePath/dist"
 $buildArtifactsPath = "$buildDirectoryPath/Artifacts"
 
+<# This is to make extension-icon.$environment.png and extension-manifest.$environment.json Linux friendly since it has a case sensitive file system#>
```

Comment:
Where does $environment come from?

Reply:
#{Octopus.Environment.Name}

Reply:
If we are okay to change them to lowercase then it may be a better option if there is no other side-effects

Reply:
Was thinking along the lines of renaming the files so they match the case of the acceptable environments?

Reply:
That's another option